melodies_monet/plots/aircraftplots.py

```diff
@@ -222,7 +229,158 @@ def add_yax2_altitude(ax, pairdf, altitude_yax2, text_kwargs, vmin_y2, vmax_y2):
+    else:
+        cmap = plt.get_cmap(grp_dict.get('color_map', 'viridis'))
+
+    norm = Normalize(vmin=vmin, vmax=vmax)
+
+    time_dates = mdates.date2num(time)
+    time_mesh, altitude_mesh = np.meshgrid(time_dates, altitude, indexing='ij')
+
+    levels = np.linspace(vmin, vmax, 200)
+    contourf_plot = axs[0].contourf(time_mesh, altitude_mesh, model_data_2d, levels=levels, cmap=cmap, norm=norm, extend='both')
+
+    # Adjust the layout to place the colorbar at the bottom
+    cbar = fig.colorbar(contourf_plot, ax=axs.ravel().tolist(), orientation='horizontal', pad=0.1, aspect=50)
+    cbar.set_label(obs_label_config.get(obs_var, {}).get('ylabel_plot', 'Variable (units)'), fontsize=12, fontweight='bold')
+    cbar.ax.tick_params(labelsize=10)
+
+    axs[0].xaxis.set_major_formatter(mdates.DateFormatter('%m-%d %H'))
+    axs[0].xaxis.set_major_locator(mdates.AutoDateLocator())
+    fig.autofmt_xdate(rotation=45, ha='right')
+
+    #axs[0].set_title(f"Model vs Observation Curtain Plot: {model_var} vs {obs_var}", fontsize=12, fontweight='bold')
+    # Set the main title and subplot titles
+    fig.suptitle(f"Model vs Observation Curtain Plot: {model_var} vs {obs_var}", fontsize=16, fontweight='bold')
+    axs[0].set_title("Model Curtain with Model Scatter Overlay", fontsize=12, fontweight='bold')
+    
+    axs[0].set_ylabel('Pressure (Pa)', fontsize=12, fontweight='bold')
+    axs[0].invert_yaxis()  # Invert y-axis to have max pressure at the bottom
+    axs[0].tick_params(axis='both', labelsize=10)
+
+    # Separate subplot for the observation scatter plot
+    axs[1].set_title("Observation Scatter", fontsize=12, fontweight='bold')
+    scatter = axs[1].scatter(time_dates, obs_pressure, c=obs_data_2d, cmap=cmap, norm=norm, edgecolor=(0, 0, 0, 0), linewidth=0.3, alpha=0.5)
+    axs[1].xaxis.set_major_formatter(mdates.DateFormatter('%m-%d %H'))
+    axs[1].xaxis.set_major_locator(mdates.AutoDateLocator())
+    fig.autofmt_xdate(rotation=45, ha='right')
+
+    axs[1].set_ylabel('Pressure (Pa)', fontsize=12, fontweight='bold')
+    axs[1].invert_yaxis()  # Invert y-axis to have max pressure at the bottom
+    axs[1].tick_params(axis='both', labelsize=10)
+    axs[1].set_xlabel('Time', fontsize=12, fontweight='bold')
+
+    # Set the same y-tick labels for both subplots and exclude the minimum value
+    y_ticks = axs[0].get_yticks()
+    axs[1].set_yticks(y_ticks)
+    axs[1].set_yticklabels([str(int(tick)) if tick != y_ticks.min() else '' for tick in y_ticks])
+
+    # Plot model scatter data on top of the model curtain plot
+    if model_scatter_data is not None:
+        axs[0].scatter(time_dates, obs_pressure, c=model_scatter_data, cmap=cmap, norm=norm, edgecolor='black', linewidth=0.5, alpha=0.7)
+
+        # Print diagnostics for model scatter data
+        print(f"model_scatter_data shape: {model_scatter_data.shape}, ndims: {model_scatter_data.ndim}")
+
+    plt.show()
```

Comment:
Can you save the plot too? Like you did for the violin plot?

Reply:
Resolved in this `[commit](https://github.com/NOAA-CSL/MELODIES-MONET/pull/254/commits/21b893789b1d8cd89a96725bfc6f5de556bbd499)`. @colin-harkins thanks for resolving this!

Reply:
![image](https://github.com/user-attachments/assets/d5420fe6-b4af-453d-a3f9-cc8f544ce187)

Reply:
![image](https://github.com/user-attachments/assets/81797cbf-db8f-47ca-93fa-d8c8c6618cf2)